### DISCUSSION TITLE
如果想要转i3那个整合好的桌面，有无快捷方法

### DISCUSSION THREAD

**Discussion by xuehrs** [ryan4yin/nixos-and-flakes-book#153] [Q&A]
我在尝试将其转为类似群晖那个Web桌面。
通过novns加Tiger VNC来实现nas系统便捷性。
可以脱离群晖了

https://nixos.wiki/wiki/Remote_Desktop

**Comment by ryan4yin** [comment]
不太理解你的需求，这是一本教 NixOS 的书，并未涉及任何图形化或 Web 相关的知识，建议去 NixOS 中文 TG 群询问这个。

**Comment by xuehrs** [comment]
OK，我之前没找到微信nixos群，这下要安装tg了。我搜索看看在哪里。谢谢2024年4月21日 下午3:35，Ryan Yin ***@***.***>写道：
不太理解你的需求，这是一本教 NixOS 的书，并未涉及任何图形化或 Web 相关的知识，建议去 NixOS 中文 TG 群询问这个。
